src/toolchain/web-guide.md

```diff
@@ -0,0 +1,135 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more to get started with compared to native builds.
```

Comment:
I think this might sound a bit more natural:

```suggestion
Web builds are a fair bit more difficult to get started with compared to native builds.
```